src/style.css

```diff
@@ -0,0 +1,81 @@
+@font-face {
+  font-family: Avenir-Heavy;
+  font-weight: 700;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  font-weight: 300px;
```

Comment:
Use font-weight with a unitless value (e.g., 300) instead of using a value with 'px'.

Reply:
done